aws/README.md

```diff
@@ -1,9 +1,22 @@
 # AWS
 
-As of this writing, we support two types of AWS deployments with Cluster API:
-- A "standard" cluster with HA control plane and a machine deployment for workers
-- A cluster that uses autoscaling groups for the worker set.
-Autoscaling groups are called "MachinePools" in Cluster API lingo.
+As of this writing, we support two types of AWS deployments with Cluster API (CAPI):
+
+- A "standard" cluster with HA control plane and a machine deployment for workers.
+- A cluster that uses autoscaling groups for the worker set. Autoscaling groups are called "MachinePools" in Cluster API lingo.
+
+We also provide different variants for Talos 0.10 and up that added better support for
+[external cloud providers](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/),
+including [cloud-provider-aws](https://github.com/kubernetes/cloud-provider-aws).
+
+| Directory                                                       | Talos | cluster-api-provider-aws (CAPA) |
+| --------------------------------------------------------------- | ----- | ------------------------------- |
+| [standard](./standard/)                                         | 0.9+  | 0.5.4                           |
+| [standard-external](./standard-external/)                       | 0.9+  | 0.5.4                           |
```

Comment:
0.5.4 is the version we actually test with: https://github.com/talos-systems/talos/blob/33035901ff7875bdf9eb99fb86b377318f60d74b/hack/test/e2e-capi.sh#L9

Should we mention it? Should we update it to 0.5.5? 0.6.4?

Reply:
I would say 0.5.x might be easier, assuming that minor versions should be fine, @rsmitty?